client/views/setupWizard/steps/OrganizationInfoStep.tsx

```diff
@@ -41,6 +44,9 @@ const OrganizationInfoStep = (): ReactElement => {
 	const organizationSizeOptions = getSettingOptions(settings, 'Size', t);
 
 	const handleSubmit: ComponentProps<typeof OrganizationInfoPage>['onSubmit'] = async (data) => {
+		if (canDeclineServerRegistration) {
```

Comment:
IMO reading this with out the context I think I'd be confused.  

`canDeclineServerRegistration` to me means the user has the ability or not to choose not to register.   But unless I'm not understanding.. that's not the case.

What this is actually controlling is **if** the registration will be skipped.  If so I'd propose: `skipCloudRegistration` or `serverAlreadyRegistered`

Something that when we come back we know what it actually means.  (same for the other occurrences)